Hot-Tip/src/HotTipAnchor.js

```diff
@@ -0,0 +1,20 @@
+import React, {useContext} from 'react'
+import {HotTipContext} from './HotTipContext'
+
+export default function HotTipAnchor() {
+  const [state] = useContext(HotTipContext)
+  const {tip, positionClass, layoutClass, visible, x, y} = state
+  const isNode = typeof tip !== 'function'
+  const className = ['hot-tip', positionClass, layoutClass].join(' ')
+
+  return (
+    Boolean(visible) && (
+      <div className={className} style={{top: y, left: x}}>
```

Comment:
Would aria alert work here? otherwise there's two options:

1.  id & described-by, this is a singleton so the id would mostly remain the same but it could be dynamic via the same context
2. HotTip would have to be structured differently to add an aria-label to it's child but this would require wrapping in a span in all cases instead of cloning the child element